client/src/partials/vouchers/complex.css

```diff
@@ -0,0 +1,3 @@
+.input-number {
```

Comment:
There is already a bootstrap class for this: `text-right` [[src](http://getbootstrap.com/css/#type-alignment)].  Your class is more semantic, but less flexible to be used in many places.

Since there is already a bootstrap class for this use case, I propose you remove this class and use bootstrap's class.